oak_containers_hello_world_trusted_app/src/orchestrator_client.rs

```diff
@@ -45,8 +35,9 @@ static IGNORED_ENDPOINT_URI: &str = "file://[::]:0";
 const IPC_SOCKET: &str = "/oak_utils/orchestrator_ipc";
 
 /// Utility struct used to interface with the launcher
+#[derive(Clone)]
 pub struct OrchestratorClient {
-    inner: GrpcOrchestratorClient<tonic::transport::channel::Channel>,
+    inner: Arc<Mutex<GrpcOrchestratorClient<tonic::transport::channel::Channel>>>,
```

Comment:
Why is this needed? The Arc shouldn't be needed becase Tonic clients are cheap to clone. The mutex should not be needed since invoking any of the methods requires a mutable reference to self, so there will be one reference to a client instance at a time.

Reply:
Done